src/components/impl/gumlet-image.js

```diff
@@ -23,8 +23,9 @@ function loadGumlet() {
 }
 
 export function GumletImage(props) {
-  const { slug, metadata, aspectRatio, imageCDN, imgParams, reactTag, className } = props;
+  const { slug, metadata, aspectRatio, imageCDN, imgParams, reactTag, className, publisherData } = props;
```

Comment:
Take only fallback image, this component doesn't require the whole publisher data